data/json/recipes/other/tool.json

```diff
@@ -177,5 +177,46 @@
     "book_learn": [ [ "mag_survival", 2 ], [ "textbook_survival", 1 ], [ "cookbook", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "can_food_unsealed", 2 ], [ "scrap", 4 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "hand_axe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 1 ],
+    "difficulty": 3,
```

Comment:
If we do allow this recipe, it should have a much higher difficulty than this. Probably 7 or higher, and fabrication 3 or so.

["We trained subjects for two years to make stone tools. It’s harder than you think! It can take hundreds of hours of dedicated practice to learn how to make a nice handaxe. It is even more difficult to figure out without some coaching."]
(https://blogs.scientificamerican.com/talking-back/learning-to-make-a-stone-age-axe-gives-clues-to-how-the-brain-evolved/?redirect=1)

Reply:
I'm kinda skeptical about all the stone tools. I just went with these ones because we already have stone axes and stuff in core. We might want to revisit the crafting requirements for all of them.